.github/workflows/docker-release.yml

```
@@ -0,0 +1,67 @@
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            BUILD_COMMIT=${{ github.sha }}
+            BUILD_DATE=${{ github.event.repository.updated_at }}
+            BUILD_BRANCH=${{ github.ref_name }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: Image digest
+        run: echo "Image digest:\ ${{ steps.meta.outputs.digest }}"
```

actionlint (1.7.9)

property "digest" is not defined in object type {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tag-names: string; tags: string; version: string}

(expression)